src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let summ = 0;
```

Comment:
Better to rename  the variable correctly (smth. like "sum") It will be nicer to read

Reply:
Ok, now its 'sum' 😄